app/lib/shared/utils.dart

```diff
@@ -173,6 +173,7 @@ final _reservedPackageNames = <String>[
   'in_app_purchase',
   'location_background',
   'webview_flutter',
+  'google_maps_flutter',
```

Comment:
maybe in alphabetical order...

Reply:
I almost want to leave them unordered as motivation for getting rid of this hack :)